### DISCUSSION TITLE
How to enable "linein"?

### DISCUSSION THREAD

**Discussion by FHFS** [sle118/squeezelite-esp32#164] [Q&A] [answered]
Hi, just built the firmware yesterday for the audiokit with esp32-a1s.
Works fine!
Is there a way to switch to the "linein" input?

**Comment by sle118** [comment, marked as answer]
That feature has not been included, I'm afraid. If anyone is willing to contribute, I'll be happy to review the code and merge as needed.

**Reply by FHFS** [reply]
atleast there is bluetooth input, I switched to using that. Probably even better than using aux. Thanks for the reply

**Reply by jamesmyatt** [reply]
Is "line in" support on the road map?

**Reply by sle118** [reply]
It is not at this point. Both Philippe and myself have little time to add new features in the short term. Maybe you could create an issue to keep it on the radar?

**Reply by nagyrobi** [reply]
> It is not at this point. Both Philippe and myself have little time to add new features in the short term. Maybe you could create an issue to keep it on the radar?

Done: https://github.com/sle118/squeezelite-esp32/issues/227
